publish/publish.proj

```diff
@@ -209,6 +209,26 @@
           DependsOnTargets="GetPackagesToSign;SignFiles">
   </Target>
 
+  <Target Name="PushSignedCoreHostPackagesToAzure">
+    <Message Text="Re-uploading signed CoreHost packages to Azure" />
+
+    <ItemGroup>
+      <ItemsToPublish Include="@(_DownloadedStandardPackages)" />
+    </ItemGroup>
+    <!-- add relative blob path metadata -->
```

Comment:
I don't think this comment provides value.

Also, these two operations don't need to be split into different `ItemGroup`s.  They can just be inside the same `<ItemGroup>`.

Reply:
Updated